ui/app/src/components/ItemStatesManagement/ItemStatesManagement.tsx

```diff
@@ -268,7 +268,7 @@ export default function ItemStatesManagement(props: ItemStatesManagementProps) {
   return (
     <section ref={rootRef} className={classes.root}>
       <GlobalAppToolbar
-        title={!embedded && <FormattedMessage id="siteTools.itemStates" defaultMessage="Item States" />}
+        title={!embedded && <FormattedMessage id="siteTools.workflowStates" defaultMessage="Workflow States" />}
```

Comment:
Check ui.xml of sites to see if this is somewhere around there too or in the defineMessages for ui config.

Reply:
Not found neither in ui.xml nor in studio-ui. 
In ui.xml the message id is `workflowStates.title`

Reply:
Let's use the same id here or there; no need for duplicate strings with the same text and target